### PR TITLE
Improve contributing docs

### PR DESCRIPTION
**Pull request by willzhang**
Improve contributing docs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,9 +115,10 @@ To put forward a PR, we assume you have registered a GitHub ID. Then you could f
 
    ```shell
    golangci-lint run -c .golangci.yml # lint
+   git add -A
    git commit -a -s -m "message for your changes" # -a is git add ., -s adds a Signed-off-by trailer
    git rebase -i	<commit-id> # do this if your pr has multiple commits
-   git push # push to your forked repository after rebase done
+   git push --set-upstream origin <new-branch> # push to your forked repository after rebase done
```
Comment (fanux) [inline CONTRIBUTING.md:121]: seems only first time needs set-upstream?

Comment (willzhang) [reply CONTRIBUTING.md:121]: OK, Then I'll delete this, and just add `git add -A`, it's ok?

Comment (fanux) [reply CONTRIBUTING.md:121]: OK, or add some describe

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -128,10 +129,11 @@ To put forward a PR, we assume you have registered a GitHub ID. Then you could f
    # create new branch, for example git checkout -b feature/infra
    git checkout -b <new branch>
    # update some code, feature1
+   git add -A
    git commit -m -s "init infra"
-   git push
+   git push # if it's first time push, run git push --set-upstream origin <new-branch>
    # then create pull request, and merge
-
+
```
Comment (zzjin) [inline CONTRIBUTING.md:136]:
```suggestion

```

**Review by zzjin — APPROVED** [review]

**Pull request merged**